test/haml/engine_test.rb

```diff
@@ -2105,4 +2105,4 @@ def assert_encoded_equal(expected, actual)
     assert_equal expected.encoding, actual.encoding
     assert_equal expected, actual
   end
-end
+end if RUBY_ENGINE != 'truffleruby' # truffleruby cannot run Haml
```

Comment:
> truffleruby cannot run Haml

Is it because of missing Ripper support? (https://github.com/oracle/truffleruby/issues/1712)

Reply:
Right.